test/acceptance/job_test.exs

```diff
@@ -13,4 +13,15 @@ defmodule Community.Acceptance.JobTest do
     assert flash_text =~ "Job updated"
     assert visible_page_text =~ "New Title"
   end
+
+  test "deleting a job" do
+    job = create(:job)
+
+    navigate_to job_path(@endpoint, :edit, job, token: job.token)
+
```

Comment:
I would either move the `navigate_to` up one, or combine `navigate_to` and `ignore_confirm_dialog` right now it looks a bit out of place and I am not sure it adds much clarity. What do you think?